packages/terra-date-picker/src/DatePicker.scss

```diff
@@ -0,0 +1,54 @@
+@import '~terra-mixins';
+@import '../node_modules/react-datepicker/dist/react-datepicker.min';
+
+.terra-DatePicker {
+  display: inline-block;
+  min-width: 14.643em; 
+}
+
+.terra-DatePicker-input {
+  border: 0.071em solid $terra-grey-40;
+  box-shadow: inset 0 0.143em 0.143em $terra-grey-20;
+  font-size: 1em;
+  height: 1.143em;
+  line-height: 1.143em;
+  margin: 0.357em 0;
+  padding: 0.429em 0.714em;
+  vertical-align: middle;
+
+  @if $terra-bidi {
+    [dir='ltr'] & {
+      border-radius: 0.286em 0 0 0.286em;
+    }
+
+    [dir='rtl'] & {
+      border-radius: 0 0.286em 0.286em 0;
+      margin-left: -0.071em;
```

Comment:
I don't think the terra-margin-start mixin will work for this scenario. The reason being that I want to set margin-left on the input only in 'rtl' and don't set margin-right in 'ltr'. Basically what I want is set margin-left only in 'rtl' for the input and set margin-left only in 'ltr' for the calendar button.